### ISSUE TITLE
[Q]: Sequence definition in "Enhanced Beat Tracking with Context-Aware Neural Networks"

### ISSUE THREAD

**Issue by siryog90** [CPJKU/madmom#303]
@superbock Hi Sebastian, I have a quick question regarding the sequences used for the Bidirectional RNN. 
1. Did you batch the sequences or loop through them one by one?
2. Did you treat each window of a song as sequence where each entry is a 120 dimensional vector, go through all the windows of a given song and loop to the next song?

**Comment by superbock** [comment]
I trained with standard stochastic gradient descent (SGD), i.e. looped through the sequences one by one. I am not entirely sure what you mean by "window of a song" but if you refer to frames, yes this is basically it. Each song/sequence has a certain number of frames (length in seconds * frame rate), and each frame has 120 dimensions.

HTH

**Comment by siryog90** [comment]
Perfect! Thank you.
